tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -269,7 +271,7 @@ def test_should_respond_200(self, session):
                     'dag_id': 'TEST_DAG_ID',
                     'dag_run_id': 'TEST_DAG_RUN_ID_2',
                     'end_date': None,
-                    'state': 'running',
+                    'state': 'queued',
                     'execution_date': self.default_time_2,
                     'external_trigger': True,
                     'start_date': self.default_time,
```

Comment:
Lets update the `create_test_dag_run` to be more reflective of new "reality" and not set start_date if the state is queued.